compiler/dyno/lib/resolution/Resolver.h

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef RESOLVER_H
```

Comment:
We haven't been prefixing the path for header files not in the include directory (i.e. that don't form part of the API) because there is less possibility for name conflicts.

E.g.

https://github.com/chapel-lang/chapel/blob/287940a91be0be3e226ee3ea5f025f929fe4a52b/compiler/dyno/lib/util/filesystem_help.h#L20-L21